lib/MySQL_Session.cpp

```diff
@@ -1536,6 +1536,34 @@ bool MySQL_Session::handler_special_queries(PtrSize_t *pkt) {
 			}
 		}
 	}
+	// Handle SQL_MODE with space. Issue #3863.
+	if (
+		pkt->size == strlen((char *)"SET SESSION sql_mode=' '") + 5 &&
```

Comment:
Since used twice, I think we can precompute this `strlen((char *)"SET SESSION sql_mode=' '")`, compiler will do but, but to remove duplication.